src/current/v23.1/sso-sql.md

```diff
@@ -1,158 +1,265 @@
 
-The **correspondence** between IdP identity and SQL username is crucial. This determined by [your cluster's identity mapping](#configure-your-clusters-identity-mapping).
+### Cluster Settings
 
-## Configure your cluster to accept your external identity provider
+| Cluster Setting | Description 
+|-----------------|------ 
+| `server.jwt_authentication.enabled` | Defaults to `false`, must be set to `true` to enable embedded JWT generation.
+| `server.jwt_authentication.jwks` | A list of public signing keys for allowed IdPs; must include your IdP's key. 
+| `server.jwt_authentication.issuers` | A list of accepted token issuers; must include your IdP.
+| `server.jwt_authentication.audience` | This must match `server.oidc_authentication.client_id`; refer to [Single Sign-on (SSO) for DB Console](sso-db-console.html).
+| `server.jwt_authentication.claim` | Which JWT field will be used to determine the user identity in CockroachDB; normally set either to `email`, or `sub` (subject).
+| `server.oidc_authentication.generate_cluster_sso_token.enabled` | Enables token generation; must be set to `true`.
+|`server.oidc_authentication.generate_cluster_sso_token.use_token`| Selects which part of the received OIDC credentials to display.
+|`server.identity_map.configuration`| Takes an [Identity Map configuration](#identity-map-configuration)
+| `server.sql_host` | This display value informs users the host for their SQL connections. Default: `localhost`.
+| `server.sql_port` | This display value informs users the port for their SQL connections. Default: `26257`.
 
-In order to authenticate a service account to a CockroachDB {{ site.data.products.cloud }} cluster using a JWT issuer, you must update several cluster settings in the `server.jwt_authentication` namespace, as well as the [`identity_map.configuration`](#configure-your-clusters-identity-mapping)
+### Update your cluster settings
 
-- [`enabled`](#server-jwt_authentication-enabled): JWT token authentication must be allowed on your cluster.
-- [`jwks`](#server-jwt_authentication-jwks): A list of public signing keys for allowed IdPs. You'll need to add your IdP's key.
-- [`issuers`](#server-jwt_authentication-issuers): A list of accepted token issuers. You'll need to add your IdP.
-- [`audience`](#server-jwt_authentication-audience): A list of audiences (or targets) for authentication, most relevantly, clusters.
 
 {{site.data.alerts.callout_success}}
-The required information for a given IdP is published on that IdP's `.well-known/openid-configuration` path (for example, `https://cockroachlabs.cloud/.well-known/openid-configuration` for CockroachDB {{ site.data.products.cloud }} or `https://accounts.google.com/.well-known/openid-configuration` for GCP.
+In order for this feature to work, [Single Sign-on (SSO) for DB Console](sso-db-console.html) and cluster SSO must both be configured with the same IDP.
```

Comment:
This doesn't seem like a tip? My immediate thought was in the prerequisites?

Reply:
Hmm prereq doesn't make sense to me, since it can't be true/false until after you do the tutorial... I see it as a troubleshooting tip, since it's a way someone could easily be messing things up and not seeing why. Does that read to you?